src/Components/AuthDialog/Hooks/useCountryCode.ts

```diff
@@ -0,0 +1,35 @@
+import { graphql } from "react-relay"
+import { useSystemContext } from "System"
+import { getENV } from "Utils/getENV"
+import { useQuery } from "Utils/Hooks/useQuery"
+import { useCountryCodeQuery } from "__generated__/useCountryCodeQuery.graphql"
+
+const USE_COUNTRY_CODE_QUERY = graphql`
+  query useCountryCodeQuery($ip: String!) {
+    requestLocation(ip: $ip) {
+      countryCode
+    }
+  }
+`
+
+export const useCountryCode = () => {
+  const { isLoggedIn } = useSystemContext()
+
+  const { data, loading, error } = useQuery<useCountryCodeQuery>({
```

Comment:
Really nice 👍 gonna bring this over to a few apps today

Reply:
Wondering if I should call this `useClientQuery`

Reply:
hmmm

Reply:
Yeah, i like that better actually.